public/css/styles.css

```diff
@@ -0,0 +1,253 @@
+#todoList,#todoList2 { list-style-type: none; padding: 0; }
+#todoList li { 
+    padding: 8px; display: flex; justify-content: space-between;
+    background: rgba(255, 255, 255, 0);
+    border-radius: 16px;
+    box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+    backdrop-filter: blur(2.4px);
+    -webkit-backdrop-filter: blur(2.4px);
+    border: 1px solid rgba(255, 255, 255, 0.16);
+ }
+ #todoList li :hover{ 
+    transform: translateY(-2px);
+ }
+#todoInput { 
+    margin: 10px; 
+    flex: 1;
+    border: none;
+    outline: none;
+    background-color: transparent;
+    padding: 1rem;
+    color: #6d2c00;
+    font-size: 20px;
+    max-width: 200px;
+    horiz-align: left;
+
+}
+
+@supports not selector(::-webkit-scrollbar) {
+    * {
+      scrollbar-width: auto;
+      scrollbar-color: #00000000 #ffffff;
+    }
+  }
+  
+  ::-webkit-scrollbar {
+    width: 12px;
+  }
+  
+  ::-webkit-scrollbar-thumb {
+    background: hwb(28 11% 57%);
+    border-radius: 10px;
+    border: 1px solid rgba(0, 0, 0, 0);
+  }
+  
+  ::-webkit-scrollbar-track {
+    background-color: #00000000;
+  }
+  
+  ::-webkit-scrollbar-button{
+    background-color: #ffffff00;
+    background-repeat: no-repeat;   
+    background-size: 50%;
+    background-position: center;
+   }
+
+
+body {
```

Comment:
I find this background colorful and i also like how it accomodates your container boxes and not allow the boxes to overlap the design itself, keep it up